### PR TITLE
Add boundary name option in assign_grille

### PR DESCRIPTION
**Pull request by lorenzovecchietti**

### CODE REVIEW

**Review by Samuelopez-ansys — COMMENTED** [review]
Please could you try to improve the codecov of this method? I see for example how flow_type is almost not tested

File: pyaedt/icepak.py [hunk 1]
```diff
@@ -4277,8 +4277,7 @@ def assign_free_opening(
             Default is ``"AmbientRadTemp"``.
         flow_type : int or str, optional
             Prescribed radiation flow type at the boundary. Available options are ``"Pressure"``,
-            ``"Velocity"`` and ``"Mass Flow"``.  Also, integers can be used, respectively 0,1 and 2.
-            Default is ``0``.
+            ``"Velocity"`` and ``"Mass Flow"``.
```
Comment (PipKat) [inline pyaedt/icepak.py:4280]:
```suggestion
            ``"Velocity"``, and ``"Mass Flow"``.
```
We use the Oxford comma, which says to put a comma before the comma and the "and" that precedes the last list item.

File: pyaedt/icepak.py [hunk 2]
```diff
@@ -216,6 +216,7 @@ def assign_grille(
         expternal_pressure="AmbientPressure",
         x_curve=["0", "1", "2"],
         y_curve=["0", "1", "2"],
+        boundary_name=None,
```
Comment (maxcapodi78) [inline pyaedt/icepak.py:219]: maybe just call it name

**Review by maxcapodi78 — APPROVED** [review]

File: pyaedt/icepak.py [hunk 3]
```diff
@@ -4355,19 +4348,19 @@ def assign_free_opening(
             ("Temperature", temperature),
             ("External Rad. Temperature", radiation_temperature),
         ]
-        if flow_type == 0:
+        if flow_type == "Pressure":
             props["Inlet Type"] = "Pressure"
```
Comment (MaxJPRey) [inline pyaedt/icepak.py:4352]:
```suggestion
        if flow_type == "Pressure":
            props["Inlet Type"] = flow_type
```

Comment (MaxJPRey) [reply pyaedt/icepak.py:4352]: Just to avoid typo error later for maintenance.

File: pyaedt/icepak.py [hunk 4]
```diff
@@ -4355,19 +4348,19 @@ def assign_free_opening(
...
             props["Inlet Type"] = "Velocity"
             possible_transient_properties += [
                 ("Static Pressure", pressure),
                 ("X Velocity", velocity[0]),
                 ("Y Velocity", velocity[1]),
                 ("Z Velocity", velocity[2]),
             ]
-        elif flow_type == 2:
+        elif flow_type == "Mass Flow":
             props["Inlet Type"] = "Mass Flow"
```
Comment (MaxJPRey) [inline pyaedt/icepak.py:4364]:
```suggestion
        elif flow_type == "Mass Flow":
            props["Inlet Type"] = flow_type
```

**Pull request merged**